### ISSUE TITLE
Extension is missing some shadow blocks that would make the experience better

### ISSUE THREAD

**Issue by jfo8000** [pimoroni/pxt-inkybit#9]
The drawImage block gives an error when you drag drop it
![image](https://user-images.githubusercontent.com/16273166/147393921-d411c9a1-4b75-4417-b12f-9bc36fa9dc83.png)

What we found after searching was this "big image" block that is (supposed?) to drop in the hole.  But it's not great because it only allows you to draw 10x5px which is very hard to see on the inky:bit

![image](https://user-images.githubusercontent.com/16273166/147393940-724980fb-e34b-4c59-ab70-7fa6928fae6e.png)

Its a little complicated but you can prefill those holes with shadow blocks so that people dont have to add in their own image.  But also I am wondering how valuable this draw image block is and how it could be made to be useful.   We're trying to draw an image across the screen - and while we have tiled the grid is too small to see any patterns on the inkybit.

**Comment by Gadgetoid** [comment]
Yeah drawing a full-screen image is... tricky. I don't know if there's any precedent for it in PXT/Makecode. The IDE almost needs an image picker, but then conversion to the correct bit-depth and resolution would have to happen *somewhere*.

I don't know when I'll get the time to mode-switch into PXT/Makecode land... it's been so long that I'm hilariously ill equipped to even discuss changes, much less make some, but this is on my TODO.

**Comment by jfo8000** [comment]
@Gadgetoid - The arcade pxt seems to have support for drawing a background image - it might be the one to look at first? 
![image](https://user-images.githubusercontent.com/16273166/149606528-aca699c3-6a8f-4b2b-b4f2-100cd67f6272.png)

**Comment by Gadgetoid** [comment]
I tried to integrate the background image stuff from Arcade, but I think it might be fundamentally incompatible... though to be fair I'm pretty rusty.

Sources: https://github.com/microsoft/pxt-common-packages/blob/3e54bbf9805405dd2da1930c6fea68867f59b9ce/libs/screen/fieldeditors.ts

And: 

https://github.com/microsoft/pxt-common-packages/blob/229d5bd24e36f82098461073a8dc45efaca6117e/libs/game/scenes.ts#L72-L82

Ended up with a fairly broken front-end. I think "Image" means something different in arcade versus regular micro:bit.
